Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md

```diff
@@ -6,14 +6,29 @@ createdAt: "2019-04-10T00:53:35.417Z"
 updatedAt: "2019-10-28T19:23:09.636Z"
 sidebar_position: -1
 ---
-Setting up Unity to create Quest content is actually pretty straightforward. This image covers it pretty well:
-![setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png](/img/setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png)
+![Building for Mobile instructions](/img/setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png)
 
-:::caution Requires Additional Setup
+If you're starting a brand new project, this won't take long at all. However, if you're converting a Windows platform project to an Android platform project, you will have to convert your assets appropriately. This can take quite a while for larger projects.
+
+For more details on best practices when working with dual-platform projects, read our documentation on [Cross-Platform Setup](/platforms/android/cross-platform-setup).
 
-[You need to install Unity's Android SDK](https://docs.unity3d.com/2019.4/Documentation/Manual/android-sdksetup.html). Otherwise the option won't pop up.
+## 3 Steps to get your world on VRChat for Quest
 
+### 1. Open Your Build Settings
+
+:::tip 
+Pros use the hotkeys Ctrl + Shift + B.
 :::
```

Comment:
Should this contain a link to Unity's documentation?

The shortcut is helpful, but admonitions should only be used for information that's very important.
```suggestion
### 1. Open Your Build Settings

You can access your [Build Settings](https://docs.unity3d.com/Manual/BuildSettings.html) from Unity's main menu by going to "File" -> "Build Settings". Or you can use the keyboard shortcut `Ctrl` + `Shift` + `B`.
```